src/Compilers/CSharp/Test/Symbol/Symbols/CovariantReturnTests.cs

```diff
@@ -18,40 +18,207 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
 {
     public class CovariantReturnTests : CSharpTestBase
     {
-        // https://github.com/dotnet/roslyn/issues/44206: For testing purposes, we support a compiler "feature" flag that permits us to skip
-        // the requirement for a runtime feature indicator. We should remove it once we have a runtime that
-        // we can test against that actually supports the feature.
-        private const string s_pretendRuntimeSupportsCovariantReturnsFeature = "DoNotRequireRuntimeCovariantReturnsSupport";
+        private static readonly MetadataReference
+            CorelibraryWithCovariantReturnSupport1,
```

Comment:
>CorelibraryWithCovariantReturnSupport1 [](start = 12, length = 38)

I think it is very uncommon (if used at all) in our code base to use multiple field declarators.

Reply:
Yes. Also uncommon to initialize fields by a static constructor.

---
In reply to: [452405555](https://github.com/dotnet/roslyn/pull/45824#discussion_r452405555) [](ancestors = 452405555)